### ISSUE TITLE
dataset size / audio generation 

### ISSUE THREAD

**Issue by markhanslip** [rahulbhalley/wavegan.pytorch#1]
Hi, thanks for the implementation :) 
Do you have suggestions for how long the input audio chunks should be and what size the dataset should be? 
At the moment I'm getting weird negative discriminator loss readings, which I wondered might be because of not enough / inappropriately segmented data. I was working with ~50mins of 4 second wav chunks. Also should the input audio data be downsampled to 16000?
I've only trained as far as ~5000 epochs and at the moment there are only glitches at the very beginning and end of the generated waveform with flat lines in between - is this a normal thing for early iterations that improves in later epochs, or is something wrong? 
Sorry for multiple questions! 
thanks again,
Mark
